chompfile.toml

```
@@ -50,31 +51,37 @@ run = 'husky install'
 
 [[task]]
 name = 'prepublishOnly'
-run = 'chomp test && chomp build'
+deps = [':test', ':build']
 
 [[task]]
 name = 'release'
 run = 'release-it'
 
 [[task]]
 name = 'server'
-run = 'chomp server:build && chomp server:start'
+deps = [':server:build', ':server:start']
 
 [[task]]
 name = 'server:build'
-run = "echo 'Building...' && chomp build && echo 'Build complete.'"
+deps = ['build']
```

This one could possibly stay:

```suggestion
run = "echo 'Building...' && chomp build && echo 'Build complete.'"
```